### PR TITLE
Added Reference URL to stats modal

### PR DESCRIPTION
**Pull request by djyarnell0**

### CODE REVIEW

File: src/components/modals/StatsModal.tsx [hunk 1]
```diff
@@ -142,6 +144,19 @@ export const StatsModal = ({
           <MigrationIntro handleMigrateStatsButton={handleMigrateStatsButton} />
         </div>
       )}
+      <hr className="mt-4 -mb-4 border-gray-500" />
+      <div className="mt-5 columns-2 items-center items-stretch justify-center text-center dark:text-white sm:mt-6">
+        <div className="mt-3 text-xs">{VIEW_VERSE_TEXT}</div>
+        <a href={referenceUrl}>
```
Comment (kadenbking) [inline src/components/modals/StatsModal.tsx:150]: Let's add `target='_blank'` to open this in a separate tab

**Review by kadenbking — CHANGES_REQUESTED** [review]
LGTM other than 1 minor change. We want people to stay on our site as much as possible.

File: src/components/modals/StatsModal.tsx (hunk 1 above)
Comment (djyarnell0) [reply src/components/modals/StatsModal.tsx:150]: Added! Good call, I didnt think about that.

**Review by kadenbking — APPROVED** [review]

**Comment by kadenbking** [comment]
One more tweak I'd like you to make. The game on the left is completed and the game on the right is not. Let's not show the referenceUrl until the game is over. That way there is no way to "cheat."
Screenshot 2023-02-26 at 7 29 03 PM

File: src/components/modals/StatsModal.tsx [hunk 2]
```diff
@@ -142,6 +144,19 @@ export const StatsModal = ({
           <MigrationIntro handleMigrateStatsButton={handleMigrateStatsButton} />
         </div>
       )}
+      <hr className="mt-4 -mb-4 border-gray-500" />
```
Comment (kadenbking) [inline src/components/modals/StatsModal.tsx:147]: You'll want to wrap this with a boolean like on line 92:

`{(isGameLost || isGameWon) && ( your code ) : <></>`

**Review by kadenbking — CHANGES_REQUESTED** [review]

File: src/components/modals/StatsModal.tsx (hunk 2 above)
Comment (djyarnell0) [reply src/components/modals/StatsModal.tsx:147]: The hr I have there is breaking that block for some reason. any ideas?

**Comment by djyarnell0** [comment]
@kadenbking Looks like It will hide with the button anyways so that should be taken care of, give it a look when you can!

File: src/components/modals/StatsModal.tsx (hunk 2 above)
Comment (kadenbking) [reply src/components/modals/StatsModal.tsx]: yeah, the problem is you can only have one child element inside a conditional. The solution is to wrap it all in a single tag.

**Comment by kadenbking** [comment]
> 

What you committed would have been fine, but if the hr is not needed until we show, I'd rather make all of it conditional

**Review by kadenbking — APPROVED** [review]

File: src/constants/strings.ts [hunk 3]
```diff
@@ -37,3 +37,5 @@ export const DATEPICKER_TITLE = 'Choose a past date'
 export const DATEPICKER_CHOOSE_TEXT = 'Choose'
 export const DATEPICKER_TODAY_TEXT = 'today'
 export const ARCHIVE_GAMEDATE_TEXT = 'Game date'
+
```
Comment (kadenbking) [inline src/constants/strings.ts:40]: Update line 28 to remove "Click here to"

**Review by kadenbking — CHANGES_REQUESTED** [review]

**Review by kadenbking — DISMISSED** [review]

**Comment by kadenbking** [comment]
@kenancasey all should be good now to review, squash, and merge

**Review by kenancasey — APPROVED** [review]
I tweaked a couple of small text things.

**Pull request merged**